src/util/records.py

```diff
@@ -106,6 +106,36 @@ def remove_view(cr, xml_id=None, view_id=None, silent=False, key=None):
         for [v_id] in cr.fetchall():
             remove_view(cr, view_id=v_id, silent=silent, key=xml_id)
 
+    # Occurrences of xml_id in the t-call of views are to be found and removed.
+    if xml_id != "?":
+        arch_col = (
+            get_value_or_en_translation(cr, "ir_ui_view", "arch_db")
+            if column_exists(cr, "ir_ui_view", "arch_db")
+            else "arch"
+        )
+        cr.execute(
+            """
+                SELECT iv.id,imd.module,imd.name
+                  FROM ir_ui_view iv
+             LEFT JOIN ir_model_data imd
+                    ON iv.id = imd.res_id AND imd.model = 'ir.ui.view'
+                 WHERE {} ~ 't-call="{}"'
+            """.format(arch_col, xml_id)
+        )
+        for vid, module, name in cr.fetchall():
+            with edit_view(cr, view_id=vid) as arch:
+                node = arch.find('.//t[@t-call="' + xml_id + '"]')
+                if node is not None:
+                    node.getparent().remove(node)
+            standard_modules = set(modules.get_modules()) - {"studio_customization"}
+            if module and module not in standard_modules or not module:
```

Comment:
Can be simplified as
```suggestion
            if not module or module not in standard_modules:
```

Reply:
Hello @KangOl, suggested changes done could you please have a look? thanks :)